### PR TITLE
Support Flutter test report

### LINKED ISSUE #28: Support Flutter test report
See https://github.com/TOPdesk/dart-junitreport/blob/master/tool/example.jsonl as a test report example.

### PR DESCRIPTION
**Pull request by edigaryev**
Resolves #28.

### CODE REVIEW

File: parsers/flutter.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+				break
+			}
+
+			return err, nil
+		}
+
+		// Does it look like a TestStartEvent[1] with a valid file associated with it?
+		//
+		// [1]: https://github.com/dart-lang/test/blob/master/pkgs/test/doc/json_reporter.schema.json
+		if entry.Type != "testStart" || entry.Test.URL == "" {
```
Comment (fkorotkov) [inline parsers/flutter.go:46]: Should we look save the test information here and only on entries like `"testID":6,"result":"success"` should we create an annotation?

Here it seems creates a warning annotation for a successful test in the future. Check out this failing task for an example of the report containing failures https://cirrus-ci.com/task/5093820242067456

Comment (edigaryev) [reply parsers/flutter.go]: Indeed, thanks, see https://github.com/cirruslabs/cirrus-ci-annotations/pull/29/commits/487b5a8485ff450a173b1a646605457b8b39e95a.

File: parsers/flutter.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+		}
+
+		// Does it look like a TestStartEvent[1] with a valid file associated with it?
+		//
+		// [1]: https://github.com/dart-lang/test/blob/master/pkgs/test/doc/json_reporter.schema.json
+		switch entry.Type {
+		case "testStart":
+			runningTests[entry.Test.ID] = entry.Test
+		case "testDone":
+			if entry.Result != "success" {
```
Comment (fkorotkov) [inline parsers/flutter.go:69]: Let's check for `error` or `failure` here then. I don't know what will happen if a test will be `skipped`. 🤔

File: parsers/flutter.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	Type   string
+	TestID int64
+	Result string
+	Test   flutterTest
+}
+
+func annotationFromTest(test flutterTest) model.Annotation {
+	return model.Annotation{
+		Type:    model.TestResultAnnotationType,
+		Level:   "warning",
```
Comment (fkorotkov) [inline parsers/flutter.go:29]: Should it be error here?

File: parsers/flutter.go (hunk 2 above)
Comment (edigaryev) [reply parsers/flutter.go:69]: >I don't know what will happen if a test will be `skipped`. 🤔

https://github.com/dart-lang/test/blob/2aabc0d096e46da512d63e9cafb3da14a396ab12/pkgs/test/doc/json_reporter.schema.json#L81

File: parsers/flutter.go (hunk 3 above)
Comment (edigaryev) [reply parsers/flutter.go]: Changed in https://github.com/cirruslabs/cirrus-ci-annotations/pull/29/commits/9a36da8fca1b908f39536cc027d5eca1910600f5.

**Review by fkorotkov — APPROVED** [review]
Once tests are passing

**Pull request merged**